depends/install_imagequant.sh

```diff
@@ -1,15 +1,37 @@
 #!/bin/bash
 # install libimagequant
 
-archive=libimagequant-4.2.2
+archive_name=libimagequant
+archive_version=4.2.2
 
-./download-and-extract.sh $archive https://raw.githubusercontent.com/python-pillow/pillow-depends/main/$archive.tar.gz
+archive=$archive_name-$archive_version
 
-pushd $archive/imagequant-sys
+if [[ "$GHA_LIBIMAGEQUANT_CACHE_HIT" == "true" ]]; then
 
-cargo install cargo-c
-cargo cinstall --prefix=/usr --destdir=.
-sudo cp usr/lib/libimagequant.so* /usr/lib/
-sudo cp usr/include/libimagequant.h /usr/include/
+    # Copy cached files into place
+    sudo cp ~/cache-$archive_name/libimagequant.so* /usr/lib/
+    sudo cp ~/cache-$archive_name/libimagequant.h /usr/include/
 
-popd
+else
+
+    # Build from source
+    ./download-and-extract.sh $archive https://raw.githubusercontent.com/python-pillow/pillow-depends/main/$archive.tar.gz
+
+    pushd $archive/imagequant-sys
+
+    time cargo install cargo-c
+    time cargo cinstall --prefix=/usr --destdir=.
```

Comment:
Yep, forgotten debugging.

```suggestion
    cargo install cargo-c
    cargo cinstall --prefix=/usr --destdir=.
```